src/modules/nodejs.rs

```diff
@@ -12,7 +12,13 @@ use crate::utils;
 pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     let is_js_project = context
         .try_begin_scan()?
-        .set_files(&["package.json"])
+        .set_files(&[
+            "package.json",
+            "package-lock.json",
```

Comment:
Each of these additions increases the time needed to gather context before printing the prompt.
I'm thinking that the lockfiles might not be necessary (how often do you have `package.json` without a lockfile?)

Reply:
How much time do you think it would add? I feel like a filesystem operation like that should be super fast, but you're the expert here.